solace-message-router/azuredeploy.json

```diff
@@ -86,38 +157,71 @@
     "imagePublisher": "OpenLogic",
     "imageOffer": "Centos",
     "OSDiskName": "osdiskfordockersimple",
+    "dataDiskName": "datadiskfordockersimple",
     "nicName": "myVMNicD",
-    "scriptUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/solace-message-router/",
-    "sharedTemplatename": "shared-resources.json",
-    "solaceInstallScriptName": "deploy_vmr.sh",
+    "baseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/solace-message-router/",
```

Comment:
Let's update this for the BPs for the repo - ee [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
Updated, introduced `_artifactsLocation` and `_artifactsLocationSasToken` parameters